### PR TITLE
Update the content

### PR DESCRIPTION
**Pull request by bojan**
The content is extremely outdated.

### CODE REVIEW

**Comment by lenart** [comment]
This is related to #1 and #2

**Comment by bojan** [comment]
@lenart @JureCindro Fri at 8:30am?

File: index.html [hunk 1]
```diff
@@ -76,28 +76,14 @@ <h1>Ziga Besal <br><span class="subtitle">Mobile Developer (Android & iOS)</span
...
-
-                    <ul class="experience-summary-list">
-                        <li>Big startup enthusiast, worked at multiple startups & lead some of them, active member of local community <a target="blank" href="https://www.facebook.com/StartupNovomesto/">Startup Novo mesto</a></li>
-                        <li>Apple WWDC Scholarship Winner</li>
-                        <li>Many years of experience with web & mobile development, familiar with modern development tools & processes for creating superb products</li>
-                        <li>Strong knowledge of backend experience in Python using Google App Engine framework, also worked with Django & PHP in the past</li>
-                        <li>Ex community/social media manager & lecturer at <a target="_blank" href="https://www.smartninja.org">SmartNinja</a></li>
-                    </ul>
-
+                    <p>I'm an experienced & skilled beeper from Slovenia, currently working at <a target="_blank" href="http://beeping.si/">Beeping</a>.</p>
```
Comment (dbreskvar) [inline index.html]: Much better description 👍

File: index.html [hunk 2]
```diff
@@ -76,28 +76,14 @@ <h1>Ziga Besal <br><span class="subtitle">Mobile Developer (Android & iOS)</span
...
                     <br>
                     <p>Visit the links above to learn more about me and stuff I do or <a href="mailto:besal.ziga@gmail.com">send me an email</a>.</p>
```
Comment (dbreskvar) [inline index.html]: I think this could change to `a href="mailto:besal.ziga@beeping.com"`

**Comment by ekranac** [comment]
run tests

File: index.html (hunk 1 above)
Comment (lenart) [reply index.html]: :shipit:

File: index.html (hunk 2 above)
Comment (lenart) [reply index.html]: Because of the high demand for beeping users I'd consider using some form of email obfuscation.

Comment (bojan) [reply index.html]: @lenart how about we create a form instead of using mails? Do you think the backend can handle the traffic?

**Pull request closed without merging**